CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features
 
+* Added `IntervalMetadata` object in `skbio.metadata._interval`. [#1356](https://github.com/biocore/scikit-bio/issues/1356)
```

Comment:
Can you change the path to `skbio.metadata`, and also note the ``Interval`` class? (Paths with an underscore are private, so shouldn't be presented to users.)